common/trace.c

```diff
@@ -38,7 +39,7 @@ static struct span *active_spans;
 static struct span *current;
 static size_t last_span_id;
 
-static void trace_span_id_serialize(struct span *s, char *dest)
+static void trace_span_id_serialize(struct span *s, char *dest)g
```

Comment:
there is a `g` at the end of the `static void trace_span_id_serialize(struct span *s, char *dest)`

Maybe we should enable the tracing inside the CI? I could help with some timeout debugging?